### PR TITLE
[lg-2] Add user and requestId logging

### PR DESCRIPTION
**Pull request by DanwerOK**

### CODE REVIEW

File: src/main/java/com/ponodan/movieland/web/intercaptor/AuthInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package com.ponodan.movieland.web.intercaptor;
```
Comment (Ramoid) [inline src/main/java/com/ponodan/movieland/web/intercaptor/AuthInterceptor.java]: intercaptor?typo?

File: src/main/java/com/ponodan/movieland/web/intercaptor/LoggerInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import com.ponodan.movieland.security.model.UserToken;
+import org.slf4j.MDC;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.UUID;
+
+public class LoggerInterceptor extends HandlerInterceptorAdapter {
```
Comment (Ramoid) [inline src/main/java/com/ponodan/movieland/web/intercaptor/LoggerInterceptor.java]: А если он перед авторизационным интерсептором запустится?

File: src/main/java/com/ponodan/movieland/web/intercaptor/AuthInterceptor.java (hunk 1 above)
Comment (DanwerOK) [reply src/main/java/com/ponodan/movieland/web/intercaptor/AuthInterceptor.java]: My bad. Fixed

File: src/main/java/com/ponodan/movieland/web/intercaptor/LoggerInterceptor.java (hunk 2 above)
Comment (DanwerOK) [reply src/main/java/com/ponodan/movieland/web/intercaptor/LoggerInterceptor.java]: Please notice, if multiple Spring interceptors are configured, the preHandle() method is executed in the order of configuration, whereas postHandle() and afterCompletion() methods are invoked in the reverse order.

http://www.baeldung.com/spring-mvc-handlerinterceptor

File: src/main/java/com/ponodan/movieland/security/impl/SecurityServiceImpl.java [hunk 3]
```diff
@@ -25,6 +25,18 @@ public SecurityServiceImpl(UserTokenCache userTokenCache) {
         this.userTokenCache = userTokenCache;
     }
 
+    private final ThreadLocal<UserToken> userTokenThreadLocal = new  ThreadLocal<>();
```
Comment (Jokeroff) [inline src/main/java/com/ponodan/movieland/security/impl/SecurityServiceImpl.java]: could be static

Comment (DanwerOK) [reply src/main/java/com/ponodan/movieland/security/impl/SecurityServiceImpl.java]: :1234:

Comment (DanwerOK) [reply src/main/java/com/ponodan/movieland/security/impl/SecurityServiceImpl.java]: Let's assume that I've added meaningful comment

**Pull request merged**